config/initializers/mapit.rb

```diff
@@ -0,0 +1 @@
+MAPIT_BASE_URL = "#{Plek.find('mapit')}/"
```

Comment:
Using gds-api-adapters means that we can instantiate the adapter here, instead of just getting the URL. Some apps (eg [smart-answers](https://github.com/alphagov/smart-answers/blob/master/lib/services.rb), [publisher](https://github.com/alphagov/publisher/blob/master/lib/services.rb), [collections-publisher](https://github.com/alphagov/collections-publisher/blob/master/lib/services.rb)) seem to be using a `Services` module now instead of an initializer for this, which we could do (although we only have mapit for now).